databases/week1/lesson1.sql

```diff
@@ -0,0 +1,20 @@
+SELECT * FROM task;
+-- Find out how many tasks are in the task table --
+SELECT COUNT(id)FROM task
+WHERE due_date IS NULL;
+-- Find all the tasks that are marked as done --
+SELECT * FROM task
+WHERE status_id = 3;
+SELECT * FROM status;
+SELECT * FROM task
+WHERE status_id IS NULL;
```

Comment:
here we need to find the task if their status NOT (done), so that means when the status is (not started) or (in progress)